source/basic.tex

```diff
@@ -991,7 +991,7 @@
 
 \begin{codeblock}
 typedef int N;
-template<N X, typename N, template<N Y> class T> struct A;
+template<N X, class N, template<N Y> class T> struct A;
```

Comment:
@zygoloid has said elsewhere that the core parts of the standard intentionally use a mix of formatting styles to reinforce the point that whitespace (or the  choice of `class` or `typename` here) has no semantic significance. So changing things to be consistent breaks that.